extensions/moji/src/index.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  randomId,
+} from "@raycast/api";
+import { useState, useEffect, useRef } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.title}
+      accessoryTitle={searchResult.name}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <OpenInBrowserAction title="Open in Browser" url={searchResult.url} />
```

Comment:
```suggestion
            <OpenInBrowserAction url={searchResult.url} />
```
The title is redundant since it's the same as default